review.css

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
You need to set the margin and padding of body to 0 to remove the default ones.
`* {
margin: 0;
padding: 0;
}`